lib/MusicBrainz/Script/RemoveEmpty.pm

```diff
@@ -153,12 +154,10 @@ sub run {
     }
 
     if ($self->summary) {
-        printf "%s : Found %d unused $entity%s.\n",
-            scalar localtime,
-            $count, ($count==1 ? '' : 's');
-        printf "%s : Successfully removed %d $entity%s\n",
-            scalar localtime,
-            $removed, ($removed==1 ? '' : 's')
+        log_info { sprintf 'Found %d unused $entity%s.',
+            $count, ($count==1 ? '' : 's') };
+        log_info { sprintf 'Successfully removed %d $entity%s',
+            $removed, ($removed==1 ? '' : 's') }
```

Comment:
Since switching doubles quotes to single quotes, `$entity` is no longer interpolated.

Reply:
Restored double quotes in commit https://github.com/metabrainz/musicbrainz-server/pull/2272/commits/a0893dff205833db03c013d73c7dff9754f5efb6.